parcel.go

```diff
@@ -14,47 +16,143 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (Client, Status, Address, Created_at) VALUES (:Client, :Status, :Address, :Created_at)",
+		sql.Named("Client", p.Client),
+		sql.Named("Status", p.Status),
+		sql.Named("Address", p.Address),
+		sql.Named("Created_at", p.Created_at))
```

Comment:
`created_at` лучше с маленькой, но тогда в 19 строчке тоже нужно поменять
```suggestion
		sql.Named("created_at", p.CreatedAt))
```